.github/workflows/ai-service-dev-ci.yml

```
@@ -0,0 +1,109 @@
+name: AI Service Dev CI/CD
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)